packages/forms/src/elements/currency/currency.tsx

```diff
@@ -80,24 +81,33 @@ const InputCurrency: React.FC<InputCurrencyProps> = ({
 			dot ? e.preventDefault() : setDot(true);
 			return true;
 		}
-		// if the input has reached the maximum length
-		if (e.target.value.length >= maxInputLength) {
-			// only allow the validKeys
-			!validKeys.includes(e.key) && e.preventDefault();
-		} else {
-			if (!digits.includes(e.key) && !validKeys.includes(e.key))
-				e.preventDefault();
+		// only allow valid keys
```

Comment:
It's only a small point, but comments could be replaced by methods with descriptive names, e.g.  here:

if (KeyPressIsNotAllowed(e)) {
   e.preventDefault();
}

Reply:
I'll change it to the proposed method.